core/src/env/srml/srml_only/sys.rs

```diff
@@ -54,6 +54,9 @@ extern "C" {
     /// This must only be used in `--dev` chain environments!
     pub fn ext_println(str_ptr: u32, str_len: u32);
 
+    /// Deposits an event
+    pub fn ext_deposit_event(data_ptr: u32, data_len: u32);
```

Comment:
Please improve the description and for comments please respect our (to be honest) very pedantic contribution guidelines: https://github.com/Robbepop/pdsl/blob/master/CONTRIBUTING.md